examples/simple-file-server/README.md

```diff
@@ -0,0 +1,74 @@
+# Simple File Server
+
+## Overview
+
+This guide explains how to put and get a file to/from a UDP server using Ballerina.
+The following figure illustrates a high-level design diagram of the complete use case.
+
+![Sending a File with UDP](./simple-udp-server.jpg)
```

Comment:
Shall reduce the image size by one third.

Reply:
Changed.